### PR TITLE
make:Fix c header prototypes

### PR DESCRIPTION
**Pull request by edwinzrodriguez**

### CODE REVIEW

File: examples/Makefile [hunk 1]
```diff
@@ -12,6 +12,8 @@ ifneq ($(USE_RTTI), 1)
 	CXXFLAGS += -fno-rtti
 endif
 
+CFLAGS += -Wstrict-prototypes
+
```
Comment (mrambacher) [inline examples/Makefile:16]: Should a comparable flag also be added for CMAKE?

Comment (edwinzrodriguez) [reply examples/Makefile:16]: I've added this flag to Makefile as well.  This should prevent future changes that break strict prototypes

Comment (edwinzrodriguez) [reply examples/Makefile:16]: Also added to CMakeLists.txt and verified passing with cmake -Wdev

**Review by ajkr — APPROVED** [review]
LGTM, test failures appear unrelated

**Pull request closed without merging**